roles/nv_gpu/tasks/ci_checks.yml

```diff
@@ -67,7 +67,9 @@
       | base64 -d
       | gunzip
       | grep dcgm
+    register: nv_dcgm_exporter_prom
```

Comment:
this was not declared on the PR definition

Reply:
fixed, I pushed it later